### PR TITLE
t4

### PR DESCRIPTION
**Pull request by JelenaCernysova**

### CODE REVIEW

File: cypress/objects/clientsPage.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        cy.visit('/clients');
+    }
+
+    pressCreateNewButton(){
+        cy.get('[class="btn"]').contains('Create Client').click();
+    }
+
+    assertClientCreated(organization){
+        this.getFirstSearchField().type(organization + '{enter}')
+        cy.get('[class="ag-cell ag-cell-not-inline-editing ag-cell-no-focus ag-cell-value"]').contains('LastName, FirstName')
```
Comment (DavidV44) [inline cypress/objects/clientsPage.js]: can't this selector be simpler?

Comment (JelenaCernysova) [reply cypress/objects/clientsPage.js]: I did not find a simpler one to use

Comment (DavidV44) [reply cypress/objects/clientsPage.js]: `.ag-cell` is a simpler one

Comment (JelenaCernysova) [reply cypress/objects/clientsPage.js]: I have already changed it to look for the correct title, or is this a better approach?

Comment (DavidV44) [reply cypress/objects/clientsPage.js]: well, this is a shorter selector that selects the same element as yours. It's worth mentioning that shorter selectors are not always better.

Comment (DavidV44) [reply cypress/objects/clientsPage.js]: In any case i thunk your selector is ok for this test, but if you ever see a very long selector try to find ways to make it simpler.

**Review by DavidV44 — APPROVED** [review]

**Pull request merged**